src/main/java/com/example/ajaxonboardingproject/nats/NatsCinemaHallGetAllController.kt

```diff
@@ -24,7 +24,7 @@ class NatsCinemaHallGetAllController(
 
     @Suppress("UnusedParameter")
     override fun generateReplyForNatsRequest(
-        request: CinemaHallRequest
+        request: Mono<CinemaHallRequest>
     ): Mono<ListOfCinemaHalls> {
         return service.getAll()
```

Comment:
What about expression body to make it look even more fast

Reply:
what you mean "fast"?

Reply:
It’s just phrase to describe expression body beauty

Reply:
Remind me about this discussion on our next daily please

Reply:
I have taken it into consideration but decided to leave it as it was, but I appreciate your advice.